src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs

```diff
@@ -1451,7 +1451,7 @@ public class DriveMatchingCoreCommandBase : CoreCommandBase
                     }
                     else
                     {
-                        if (nameMatcher.IsMatch(drive.Name))
+                        if (nameMatcher != null && nameMatcher.IsMatch(drive.Name))
```

Comment:
Can this be 
```suggestion
                        if (nameMatcher?.IsMatch(drive.Name))
```

Reply:
Report is "Cannot implicitly convert type bool? to bool."

Reply:
Maybe this works?
```CSharp
if (nameMatcher?.IsMatch(drive.Name) ?? false)
{
}
```

Reply:
Looks as puzzle :-)